sequencing/ATACSeq/preprocessing/alignment.sh

```diff
@@ -1,43 +1,48 @@
-## Takes trimmed reads for single sample aligns with bowtie2 and filters out duplicates, mt reads, incorrectly paired reads
-
-## REQUIRES the following variables in config file
-# RAWDATADIR, TRIMDIR, ALIGNEDDIR, REFGENOME, multimap
-
-## REQUIRES the following software
-# bowtie2, samtools, picard
-
-## INPUT
-# 2 trimmed fastq files
-
-## OUTPUT
-# *_statsperchr.txt
-# *_sorted.bam
-# *_sorted_chr1.bam
-# *_dupMetrics.txt
-# *_depDup_q30.bam
-# *_postFilter_statsperchr.txt
-
-## (command line) VARIABLES
+## ===================================================================================================================##
```

Comment:
I know this script is being ran using another one. But just in case the script is ran on its own in the future, could you put a shebang line for this script as well?